W 11815997 rpa builder workbench cv (#84)
mulesoft/docs-rpa: rpa-builder/6.0/modules/ROOT/pages/workbench-elements-in-detail.adoc

= Workbench Elements in Detail

The Workbench contains a graphical representation of a Workflow. The Workflow type shows as a watermark in the upper right corner.
The Workbench is capable of holding several windows as tabs, which enables you to work on more than one project at a time. Hold the mouse pointer over a tab for a short time and the path to the folder where the project is stored shows as a tooltip.

image::workbench-tooltip.png[The Workbench showing the location path tooltip, 80%, 80%]
[calloutlist]
. Multiple Workflows organized in tabs
. Project location tooltip
. Workflow type icon

== Selecting Objects

To select an object, click on its icon. Selected objects show a blue border and their properties show in the Properties panel. Right-click to access a context menu for any Workbench element:

image::workbench-context-menu2.png[The context menu of a Workbench element, 50%, 50%]
[calloutlist]
. The context menu
. The selected object

To select more than one object, press and hold _Ctrl_ or _Shift_ while clicking to select the objects. If you drag an object into the Workbench, it is automatically selected.

== Context Menu Options

> The following list includes the context menu elements and describes each of them.

Reviewer: Suggestion: The context menu consists of the following context menu elements:

* *Disable/Enable*
+

> Use this option to disable an Action Step and prevent its execution or to enable an Action Step that was previously disabled. Disabled Action Steps show greyed out on the workbench.

Reviewer: ```suggestion
Use this option to disable an Action Step and prevent its execution or to enable an Action Step that was previously disabled. Disabled Action Steps are greyed out on the workbench.
```

* *Enable All Selected/Disable All Selected*
+
Use this option to disable or enable multiple Action Steps simultaneously.
* *Save As Template*
+

> Saves the selected Action Step with all settings as a template which you can use again later. See xref:managing-user-templates.adoc[Managing User Templates] for additional details about creating, importing, and exporting Templates.

Reviewer: ```suggestion
Saves the selected Action Step and its settings as a template that you can use again later. See xref:managing-user-templates.adoc[Managing User Templates] for additional details about creating, importing, and exporting Templates.
```
